tests/integration/test_text_classification.py

```diff
@@ -0,0 +1,137 @@
+import json
+import random
+
+import numpy as np
+import pandas as pd
+import pytest
+import torch
+import yaml
+from biome.text import Pipeline, TrainerConfiguration, VocabularyConfiguration
+from sklearn.model_selection import train_test_split
+
+
+@pytest.fixture
+def train_valid_yaml(tmp_path):
+    def create_csv_and_return_yaml(dataframe, name):
+        csv_path = tmp_path / f"{name}.csv"
+        dataframe.to_csv(str(csv_path), index=False)
+        yaml_path = csv_path.parent / f"{name}.yml"
+        with yaml_path.open("w") as file:
+            yaml.safe_dump({"source": str(csv_path)}, file)
+
+        return yaml_path
+
+    df = pd.read_csv(
+        "s3://biome-tutorials-data/text_classifier/business.cat.10k.csv"
+    ).iloc[:2000]
+    df_train, df_valid = train_test_split(df, test_size=0.2, random_state=42)
+
+    train = create_csv_and_return_yaml(df_train, "train")
+    valid = create_csv_and_return_yaml(df_valid, "valid")
+
+    return str(train), str(valid)
+
+
+@pytest.fixture
+def pipeline_dict():
+    """Pipeline config dict. You need to update the labels!"""
+    pipeline_dict = {
```

Comment:
It would be nice to programatically create the pipeline configuration `PipelineConfiguration` and the save as a yaml file. 

We can detect errors related with attributes renaming/deleting or refactoring

Reply:
> It would be nice to programatically create the pipeline configuration PipelineConfiguration and the save as a yaml file.

This was my first approach, but i did not manage. I will try again today.